NEWS.md

```diff
@@ -79,6 +79,15 @@ Breaking changes:
   conventional LP interpretations of dual values as sensitivities of the
   objective value to relaxations of constraints.
 
+- `@constraintref` is no longer defined. Instead, create the appropriate
+  container to hold constraint references manually. For example,
+  ```julia
+  constraints = Dict() # Optionally, specify types for improved performance.
```

Comment:
I will probably add here the example of using the JuMPArrays instead of `dict()`if the interest is to have users use the JuMPArrays more commonly than the axis arrays.

Reply:
`Dict()` is a simpler drop-in replacement for 90% of previous use cases of `@constraintref`. There's no specific intent to push JuMPArrays everywhere possible.